website/docs/r/cdn_frontdoor_route.html.markdown

```diff
@@ -0,0 +1,179 @@
+---
+subcategory: "CDN"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cdn_frontdoor_route"
+description: |-
+  Manages a Frontdoor Route.
+---
+
+# azurerm_cdn_frontdoor_route
+
+Manages a Frontdoor Route.
```

Comment:
should these be "CDN FrontDoor" here for consistency across resources?

Reply:
Fixed.